### ISSUE TITLE
Ionic CLI 3.16.0 Livereload missing server commands

### ISSUE THREAD

**Issue by omargon** [ionic-team/ionic-cli#2872]
**Current behavior:**
We just upgraded our Ionic CLI from 1.7.16 to the latest 3.16.0 and we noticed that we are missing the usual ionic server command options when running `ionic cordova emulate {device} -l`

**Expected behavior:**

We use to run `ionic emulate {device} -l` and get the following server options:
```
  restart or r to restart the client app from the root
  goto or g and a url to have the app navigate to the given url
  consolelogs or c to enable/disable console log output
  serverlogs or s to enable/disable server log output
  quit or q to shutdown the server and exit
```

**Ionic info:**
```
cli packages: (/Users/{user}/.nvm/versions/node/v6.11.4/lib/node_modules)

    @ionic/cli-utils  : 1.16.0
    ionic (Ionic CLI) : 3.16.0

global packages:

    cordova (Cordova CLI) : 7.1.0

local packages:

    Cordova Platforms : android 6.3.0 ios 4.5.2
    Ionic Framework   : ionic1 1.3.1

System:

    ios-deploy : 1.9.1
    ios-sim    : 6.1.2
    Node       : v6.11.4
    npm        : 3.10.10
    OS         : macOS Sierra
    Xcode      : Xcode 9.1 Build version 9B55

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro
```

**Comment by dwieeb** [comment]
@omargon Yes, they were not added back when the CLI was rewritten. Were these particularly useful?

**Comment by omargon** [comment]
@dwieeb Yes, at least in our current workflow every time we made a change to a JS(controller) file we have to restart the simulated app due to session variables cleared out and a blank screen follow by it. Without being able to restart from terminal we have to completely close the app in the emulator, re-open it and start a new developer tools session as well.

**Comment by dwieeb** [comment]
@omargon That's a great reason! But doesn't live reload accomplish this? Is there something about live reload that isn't working? Live reload should restart the app in the same way `r` or `restart` did.

**Comment by omargon** [comment]
@dwieeb as far as I know Livereload didn't and doesn't restart the client app from root, just refreshes the modified files and refreshes the current view/url/route. When modifying CSS or HMTL files the view gets automatically refreshed without crashing the app. But when a JS gets modified refreshing the current state/route crashes our app, so we are forced to `restart` via ionic CLI terminal command or by killing the app in the emulator. If there is a parameter or a way to force Livereload to restart the app from the root we will definitely use it.

**Comment by dwieeb** [comment]
@omargon By "from root" you mean from the app root at `/` instead of a custom view/url/route?

Do you think the reason it crashes your app is a common issue? Reason I ask is because this is feature has been requested before, but only once. I wasn't aware of the need.

**Comment by omargon** [comment]
@dwieeb Yes, from root meaning / but not sure if anything else happens when the server `restart` was executed but I believe that the app was restarted completely, session variables and all. I do reference root because that was the description given when running the `ionic emulate {device} -l` we used to get with it.
```
  restart or r to restart the client app from the root
  goto or g and a url to have the app navigate to the given url
  consolelogs or c to enable/disable console log output
  serverlogs or s to enable/disable server log output
  quit or q to shutdown the server and exit
```

**Comment by dwieeb** [comment]
What do you mean by "session variables"? `sessionStorage`?
